### ISSUE TITLE
Chrome shortcuts not per app

### ISSUE THREAD

**Issue by talleux** [dmarmor/epichrome#251]
After playing around with some tab extensions that set some keyboard shortcuts (like tab-snooze or onetab) i discovered that these shortcuts applied when any SSBs/apps is in focus, even is they don't have the extension installed (and shortcuts are not set as global but only in chrome. To make it clear (how to reproduce):
- App A has an extension with a shortcut set
- App B has NOT thins extension installed and is in the focus
- Press the shortcut set in App A --> This will bring App A in the focus and apply the shortcut

I had to remove the shortcuts in all (!) the Apps that had them set in order to get rid of this.
I didn't try to set different shortcuts per App, which could be a productivity use case. This makes the situation probably worse...

- All Apps are on Version 2.3.2
- All Apps are based on Chrome

Any idea if this can be fixed? Can i give more infos?

**Comment by dmarmor** [comment]
I'm so far unable to reproduce this. I installed OneTab, which appears to set an In-Chrome shortcut of Opt-Shift-1 to display its window. As long as the shortcut was set to `In Chrome` then it would not do anything when another Chrome-based app was in front. And if I installed the extension in multiple apps (and in Chrome itself) with the same shortcut set in each, the OneTab window always opened in the proper app.

Now, if I set the shortcut to `Global` in both apps, then hitting it would cause the OneTab window to open in *both* apps. If it was `Global` in only one app, then the OneTab window would open only in the one where it was set `Global`. All this seemed to me like expected behavior.

I did notice that TabSnooze has one shortcut set `Global` by default (Cmd-Shift-1 for "New todo tab). Is it possible you're seeing the behavior with this shortcut?

If not, and you're sure you're seeing it with `In Chrome` shortcuts, then would it be possible to do a screen recording to show me the behavior (also showing how the shortcut is set on the app's Extensions page)?
